patches/tModLoader/Terraria/ModLoader/PrefixLoader.cs

```diff
@@ -5,6 +5,7 @@
 using Terraria.ID;
 using Terraria.ModLoader.Core;
 using Terraria.Utilities;
+using static Terraria.GameContent.UI.EmoteID;
```

Comment:
Accidental?

Reply:
Yes, I I'd guess I must've accidentally autocompleted something that added that and not noticed.